custom-recipes/object-detection-query-sampler/recipe.py

```diff
@@ -0,0 +1,461 @@
+import os
+import json
+import logging
+import random
+
+import dataiku
+from dataiku.customrecipe import *
+
+import pandas as pd
+import numpy as np
+import tensorflow as tf
+
+from sklearn.model_selection import train_test_split
+from keras.utils import multi_gpu_model
+from keras.models import load_model
+
+from keras_retinanet.models.resnet import resnet50_retinanet
+from keras_retinanet.models.retinanet import *
+from keras_retinanet.utils.model import freeze as freeze_model
+from keras_retinanet.utils.image import read_image_bgr, preprocess_image, resize_image
+from keras_retinanet import backend
+from keras_retinanet.layers import FilterDetections
+from lal import utils
+from cardinal import uncertainty
+
+
+
+def prettify_error(s):
```

Comment:
This code is duplicated 3 times: in 2 recipes and in the helper. Let's maybe put 1 version to utils.py and import it from there?

Reply:
Fixed.